docs/projects/ad9656_fmc/index.rst

```diff
@@ -0,0 +1,287 @@
+   :header-rows: 1
+
+   * - SPI type
+     - SPI manager instance
+     - SPI subordinate
+     - CS
+   * - PS
+     - SPI 0
+     - AD9656
+     - 1
+   * - PS
+     - SPI 0
+     - AD9508
+     - 1
+   * - PS
+     - SPI 0
+     - AD9953
+     - 1
+
+Interrupts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Below are the Programmable Logic interrupts used in this project.
+
+.. list-table::
+   :widths: 30 10 15 15
+   :header-rows: 1
+
+   * - Instance name
+     - HDL
+     - Linux ZynqMP
+     - Actual ZynqMP
+   * - axi_ad9656_rx_jesd
+     - 12
+     - 108
+     - 140
+   * - axi_ad9656_rx_dma
+     - 13
+     - 109
+     - 141
+
+Building the HDL project
+-------------------------------------------------------------------------------
+
+The design is built upon ADI's generic HDL reference design framework.
+ADI distributes the bit/elf files of these projects as part of the
+:dokuwiki:`ADI Kuiper Linux <resources/tools-software/linux-software/kuiper-linux>`.
+If you want to build the sources, ADI makes them available on the
+:git-hdl:`HDL repository </>`. To get the source you must
+`clone <https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository>`__
+the HDL repository.
+
+Then go to the project location (**projects/ad9656_fmc/zcu102**) and run the
+make command by typing in your command prompt:
+
+**Linux/Cygwin/WSL**
+
+.. code-block::
+
+   user@analog:~$ cd hdl/projects/ad9656_fmc/zcu102
+   user@analog:~/hdl/projects/ad9656_fmc/zcu102$ make
+
+The following dropdown contains a table with the parameters that are used to
+configure this project, on :xilinx:`ZCU102`.
+
+.. collapsible:: Default values of the ``make`` parameters for AD9656-FMC
+
+   +-------------------------+--------------------------------+
+   | Parameter               | Default value of the parameters|
+   |                         |            depending on carrier|
+   |                         +--------------------------------+
+   |                         | ZCU102                         |
+   +=========================+================================+
+   | JESD_MODE               | :red:`8B10B`                   |
+   +-------------------------+--------------------------------+
+   | RX_NUM_OF_LANES         | 4                              |
+   +-------------------------+--------------------------------+
+   | RX_NUM_OF_CONVERTERS    | 4                              |
+   +-------------------------+--------------------------------+
+   | RX_SAMPLES_PER_FRAME    | 1                              |
+   +-------------------------+--------------------------------+
+   | RX_SAMPLE_WIDTH         | 16                             |
+   +-------------------------+--------------------------------+
+   | RX_SAMPLES_PER_CHANNEL  | 2                              |
+   +-------------------------+--------------------------------+
+
+A more comprehensive build guide can be found in the :ref:`build_hdl` user guide.
+
+Resources
```

Comment:
There is a warning with the sections from Resources. Hint: that is because from the beginning of the file you used the wrong symbols for the other subsections